app/services/hts_service/ait_intergration/contact_csv_row_builder.rb

```diff
@@ -10,7 +10,15 @@ def last_name contact, index
     end
 
     def sex contact, index
-      contact['Gender of contact']
+      case contact['Gender of contact']&.strip
```

Comment:
For the gender of the contact do always assumme they are not pregnant?

Reply:
resolved